cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -50,6 +50,31 @@ Replaced by:
 ----
 DROP CONSTRAINT name [IF EXISTS]
 ----
+
+
+a|
+label:syntax[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+CREATE CONSTRAINT FOR ... REQUIRE ...
+----
+a|
+New syntax for creating constraints, applicable to all constraint types.
+
+a|
+label:syntax[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CREATE CONSTRAINT ON ... ASSERT ...
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+CREATE CONSTRAINT FOR ... REQUIRE ...
+----
```

Comment:
We should also update the replacements of `CREATE CONSTRAINT ... ASSERT exists()` below to this, so users don't have to migrate in two steps